backend/rdf4j/src/main/java/org/dotwebstack/framework/backend/rdf4j/query/context/AbstractVerticeFactory.java

```diff
@@ -260,4 +268,22 @@ void addOrderables(Vertice vertice, OuterQuery<?> query, Map<String, Object> ord
     orderables.add((Objects.isNull(order) || order.equalsIgnoreCase("desc")) ? subject.desc() : subject.asc());
     vertice.setOrderables(orderables);
   }
+
+  void addLanguageFilter(Edge edge, PropertyShape propertyShape) {
+    if (Objects.equals(RDF.LANGSTRING, propertyShape.getDatatype())) {
+      edge.getObject()
+          .getFilters()
+          .add(createLanguageFilter());
+    }
+  }
+
+  private Filter createLanguageFilter() {
+    ArrayList<Operand> languages = new ArrayList<>();
+    languages.add(Rdf.literalOf(rdf4jProperties.getShape()
+        .getLanguage()));
+    return Filter.builder()
```

Comment:
Niet nodig om een aparte arraylist aan te maken:
```suggestion
   return Filter.builder()
        .operator(FilterOperator.LANGUAGE)
        .operands(ImmutableList.of(Rdf.literalOf(rdf4jProperties.getShape()
            .getLanguage())))
        .build();
```